seed/models/data_quality.py

```diff
@@ -109,9 +113,11 @@ class Rule(models.Model):
     ]
 
     SEVERITY_ERROR = 0
-    SEVERITY_WARNING = 1
+    SEVERITY_VALID = 1
```

Comment:
I think this is going to break the consistency of the database. Presently there are 1's in the database, which means that any rule that was a warning, is not a valid data check.

![image](https://user-images.githubusercontent.com/1907354/70644126-9ba34380-1bff-11ea-9160-749867ae75f3.png)

Need to change validity check to be 2. 

I'll make this change. This will also require a migration. Need to call `./manage.py makemigrations`. If you run this command you will notice that it is not going to renumber any of the Rules in the database.